digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -1166,6 +1180,15 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     )
   }
 
+  def junctionInfoToApi(junctionInfo: JunctionInfo) : Map[String, Any] = {
+      Map("id" -> junctionInfo.id,
+        "junctionNumber" -> junctionInfo.junctionNumber,
+        "nodeId" -> junctionInfo.nodeId,
+        "startDate" -> formatDateTimeToShortPatternString(Some(junctionInfo.startDate)),
```

Comment:
Just a quick note: **formatDateTimeToShortPatternString** was already define as **formatToString**.

Although I think that this implementation is better since it receives an actual DateTime.
I think we can try to use only this function on other places, to avoid two functions that do the same thing.

Then, formatToString can be removed.